### DISCUSSION TITLE
Awesome.

### DISCUSSION THREAD

**Discussion by iPoetDev** [localsend/localsend#774] [Discussion (English, International)]
I found this on the GitHub front page. Timing was just right. I have set up a new device and repurposes a second much older one. I wanted to share files, after sharing Mouse without borders keyboard and mouse, and I did not know I wanted a local file transfer. Serendipity. Thank you.

**Comment by Francesco146** [comment]
Me too!

**Comment by alex-djangsters** [comment]
How do I star this repo second time??? I really need to :D
